Testing: Update to-test.md prior to Jetpack 14.5 release (#42679)
Automattic/jetpack: projects/plugins/jetpack/to-test.md

You can see a [full list of changes in this release here](https://github.com/Automattic/jetpack-production/blob/trunk/CHANGELOG.md). Please feel free to test any and all functionality mentioned!

### Alt text and caption generation tools

The image block extension now has alt text and caption generation options via Jetpack AI.

1. In the block editor, add an image block.

> 2. You should be able to see the AI stars icon, but if there is no image, the icon must be disabled.

Reviewer: I had to read this a few times before I could parse it correctly. Maybe like this instead?
```suggestion
2. You should be able to see the AI stars icon, which should be disabled because the block has no image selected yet.
```
